src/openforms/formio/tests/test_validators.py

```diff
@@ -202,6 +202,22 @@ def test_allowed_mime_types_for_csv_files(self):
 
             validator(sample)
 
+    def test_allowed_mime_types_for_msg_files(self):
+        valid_type = "application/vnd.ms-outlook"
+        msg_file = TEST_FILES / "test.msg"
+        validator = validators.MimeTypeValidator(allowed_mime_types=[valid_type])
+
+        # 4795
+        # The sdk cannot determine the content_type for .msg files correctly.
+        # Because .msg is a windows specific file, and linux and MacOS don't know it.
+        # So we simulate the scenario where content_type is unknown
+        sample = SimpleUploadedFile(
+            "test.msg",
+            msg_file.read_bytes(),
+        )
```

Comment:
This results in the file having `content_type="text/plain"` because it's the default of `SimpleUploadedFile`. You need to explicitly reproduce the case where it's empty as provided by the frontend:

```suggestion
        sample = SimpleUploadedFile(
            "test.msg",
            msg_file.read_bytes(),
            content_type="",
        )
```